src/carExample/Car.java

```diff
@@ -15,12 +16,12 @@
     private List<CarDoors> doors = new ArrayList<>();
 
     public Car(String productionDate) {
-        this.productionDate = productionDate;
+        this.productionDate = LocalDate.now();
     }
 
     public Car(String productionDate, String engineType, int maxSpeed, double timeToHundreetKPH,
```

Comment:
same as above: String productionDate

Reply:
just inattention, sorry
already push it to master